ex3/pom.xml

```diff
@@ -0,0 +1,13 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <groupId>org.fcuenca</groupId>
+  <artifactId>ex3</artifactId>
+  <version>0.0.1-SNAPSHOT</version>
+  <packaging>pom</packaging>
+  <name>ex3</name>
+  <modules>
+  	<module>../ex1</module>
+  	<module>../ex2</module>
+  </modules>
+</project>
```

Comment:
The module part shouldn't be part of exercise. The exercise #3 stays to start adding some dependencies (libraries), adding some source code, and customizing the project to suit specific needs. 
Also, for the multi-module project exercise, it's needed to update the first pom you defined on exercise #1